### PR TITLE
Task 4. Dialog component

### PR DESCRIPTION
**Pull request by dhankah**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,37 @@
...
+import Dialog from './components/Dialog.js';
+import MovieForm from './components/MovieForm.js';
+
+import './styles/App.css';
 
 function App() {
   const initialValue = 0;
   const values = ['All', 'Documentary', 'Comedy', 'Horror', 'Crime'];
   const current = values[0];
+  var movieInfo = {
```
Comment (Imaginary05) [inline src/App.js]: Use `const` instead of `var`

File: src/components/MovieDetails.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import '../styles/MovieDetails.css';
+
+const MovieDetails = ({imageUrl, movieName, releaseYear, genres, rating, duration, description}) => {
+
+     return (
+         <div class = 'detailsContainer'>
+           <img src = {imageUrl} class = 'image'/>
```
Comment (Imaginary05) [inline src/components/MovieDetails.js]: Please check the indents.

File: src/components/MovieForm.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    const [selectedGenre, setSelectedGenre] = useState(movieInfo.genres); 
+
+    const handleGenreChange = (event) => {
+        setSelectedGenre(event.target.value); 
+    };
+
+    const genresList = ["Horror", "Drama"]
+
+     return (
+         <div class = 'movieForm'>
```
Comment (Imaginary05) [inline src/components/MovieForm.js]: Please use `form` tag

File: src/components/MovieForm.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+           <div class="column">
+           <p>movieTitle</p>
```
Comment (Imaginary05) [inline src/components/MovieForm.js]: Please use `label` instead of `p`

File: src/components/MovieTile.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import '../styles/MovieTile.css';
+
+export default function MovieTile (props) {
+
+     return (
+         <div class = 'container' onClick={props.onSelect} data-testid='movie-tile'>
+           <img src = {props.imageUrl}/>
```
Comment (Imaginary05) [inline src/components/MovieTile.js]: Please use `alt` attribute

Comment (dhankah) [reply src/components/MovieTile.js]: Thank you, updated

**Pull request merged**